common/lib/xmodule/xmodule/js/src/video/04_video_full_screen.js

```diff
@@ -165,9 +158,7 @@ define('video/04_video_full_screen.js', [], function () {
         this.videoFullScreen.fullScreenEl
             .find('.icon')
                 .removeClass('fa-arrows-alt')
-                .addClass('fa-compress')
-                .find('.control-text')
-                    .text(gettext('Exit full browser'));
+                .addClass('fa-compress');
```

Comment:
I believe the title should be changed to "Exit full browser", unless we are OK with it always saying "Fill Browser". I verified that currently the title is not changing.